packages/babel-preset-react-app/package.json

```diff
@@ -1,11 +1,11 @@
 {
```

Comment:
I need help figuring out how to publish to artifactory.

Reply:
yeah...I think that I might prefer we just pull this out completely into its own github repo rather than have it be part of the fork of create-react-app.  Here are a few reasons.

1) I doubt facebook updates this babel plugin very often, so the "ease" of upstream merges isn't a big selling point for keeping it in here.
2) By removing so much configuration and changing so much, merging upstream changes is not going to be a pleasant experience anyways, so moving to own repo is not costly in that regards
3) Setting up Travis the way we want is not going to be trivial if we keep it in here. But if we move it to its own repo, it will be.